types/index.d.ts

```diff
@@ -0,0 +1,324 @@
+/// <reference types="node" />
+
+/**
+ * Supported key types.
+ */
+export type KeyType = "Ed25519" | "RSA" | "secp256k1";
```

Comment:
Note that, these are case-sensitive. They could be made case-insensitive by adding lower-/upper-case values. This goes for all `*Type` below.